.eslintrc.yaml

```diff
@@ -256,6 +256,11 @@ rules:
   react/require-optimization: off
   react/require-render-return: error
   react/self-closing-comp: error
+  react/sort-comp: [warn, {order: [type-annotations,
+                                   static-methods,
+                                   lifecycle,
+                                   everything-else,
+                                   render]}]
```

Comment:
Instead of changing order for all files but `TagEditor.js`, would it be possible to change order for `TagEditor.js` only?

Reply:
I guess it depends on whether we have a reason to want this order or not. I didn't pick the order for the rule - @mwiencek, comments? :)

Reply:
Well, I do think the suggested order is the most natural, but I'm also okay with dropping this rule entirely: class-based components should eventually be converted to functional ones, anyway, and the order doesn't impact readability enough to enforce it IMO.

Reply:
@yvanzo: enforce the rule for now, or drop the rule for now?

Reply:
I dropped it.